.cirrus.yml

```diff
@@ -166,3 +166,41 @@ static_binary_task:
     save_cache_script: |
         mv /nix .cache
         chown -Rf $(whoami) .cache
+
+# Build the static binary for arm64
+static_binary_arm64_task:
+    depends_on:
+        - 'config'
+        - 'fmt'
+
+    gce_instance:
+        image_name: "${FEDORA_CACHE_IMAGE_NAME}"
+        cpu: 8
+        memory: 12
+        disk: 200
+
+    init_script: |
+        set -ex
+        setenforce 0
+        growpart /dev/sda 1 || true
+        resize2fs /dev/sda1 || true
+        yum -y install podman
```

Comment:
I'm not in favor of run-time installs, they tend to cause ugly (hard to solve) automation-reliability problems.  In any case, the podman package should already be installed (along with buildah and skopeo) in every VM image.  That will never change, we have to install it to make sure all default dependencies are present.  Did you find this not to be the case?

Reply:
No I just took that part from the existing static build. I'll clean that up after this PR got merged. :+1: